plugins/devtools-backend/src/api/DevToolsBackendApi.ts

```diff
@@ -223,8 +223,8 @@ export class DevToolsBackendApi {
       backstageJson = JSON.parse(buffer.toString());
     }
 
-    const lockfilePath = paths.resolveTargetRoot('yarn.lock');
-    const lockfile = await Lockfile.load(lockfilePath);
+    const pacman = await detectPackageManager();
```

Comment:
With Yarn it was always safe to assume that the lock file would be available for this feature. With these changes I assume that would still be true, but can you confirm? Also, would that be true for other package managers, is it safe to assume their lock files will be included with Backstage when deployed?

Reply:
The logic didn't change, so yes, your assumption is still true. To confirm, when using another package manager then the `yarn.lock` won't be present., so with pnpm there would be a `pnpm-lock.yaml` instead.